package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "bap-node-microframework-ping-module",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "tsc": "tsc",
+    "build": "npm run tsc && find . -type f -depth 1 -name '*.ts' | xargs -I {} tsc {} typings/index.d.ts --declaration --module commonjs",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+      "bap-node-microframework": "git+https://github.com/bap-node-microframework/node-microframework.git",
+      "bap-node-microframework-mongoose": "git+https://github.com/bap-node-microframework/node-microframework-mongoose.git"
+  }
```

Comment:
Sequelize also so should move to core
Le 7 juin 2016 10:36, "Nicola Rizzo" <notifications@github.com> a écrit :

> In package.json
> <https://github.com/bap-node-microframework/node-microframework-ping-module/pull/1#discussion_r66030112>
> :
>
> > +{
> > +  "name": "bap-node-microframework-ping-module",
> > +  "version": "1.0.0",
> > +  "description": "",
> > +  "main": "dist/index.js",
> > +  "scripts": {
> > +    "tsc": "tsc",
> > +    "build": "npm run tsc && find . -type f -depth 1 -name '*.ts' | xargs -I {} tsc {} typings/index.d.ts --declaration --module commonjs",
> > +    "test": "echo \"Error: no test specified\" && exit 1"
> > +  },
> > +  "author": "",
> > +  "license": "ISC",
> > +  "dependencies": {
> > +      "bap-node-microframework": "git+https://github.com/bap-node-microframework/node-microframework.git",
> > +      "bap-node-microframework-mongoose": "git+https://github.com/bap-node-microframework/node-microframework-mongoose.git"
> > +  }
>
> BaseControllerMongoose has router property.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/bap-node-microframework/node-microframework-ping-module/pull/1/files/58f03cd5d99dde4093ea898ee0fe913aa880b2d6#r66030112>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAnw3B0HuWRbaWQtArPaw-0S_WLNwRTDks5qJS2mgaJpZM4IvrNU>
> .
>